Assets/Scripts/Managers/GameManager.cs

```diff
@@ -60,10 +59,10 @@ public void EndGame()
 
     private void Start()
     {
-        PlayerScript p = GameObject.FindObjectOfType<PlayerScript>();
-        if(p != null)
+        Player[] p = GameObject.FindObjectsOfType<Player>();
+        for (int i = 0; i < p.Length; i++)
         {
-            player = p.gameObject;
+            player.Add(p[i]);
```

Comment:
This can be done in a single line "player.AddRange(GameObject.FindObjectsOfType<Player>());"